### ISSUE TITLE
toString on ValueChange

### ISSUE THREAD

**Issue by jrpe99** [javers/javers#577]
toString does not show the same as processChangeList with SimpleTextChangeLog
The correct change is 'null' -> '' as displayed by processChangeList , but toString displays old value as ''.

`
ValueChange valueChange = (ValueChange) change;
`
`
LOG.debug("Property name : {}", valueChange);
`
=>
property:'internAnmarkningstext', oldVal:'', newVal:''}

`processChangeList`
=>value changed on 'internAnmarkningstext' property: 'null' -> ''

**Comment by bartoszwalacik** [comment]
maybe a pull request to fix this issue?

**Comment by jrpe99** [comment]
Hi, I can't. To many things to do ... we are evaluating Javers as one possibility in a project. If the project approve us using it, then I may get the time :-)

**Comment by bartoszwalacik** [comment]
This issue is not clear for me.  We have simple and low level toString()
and pretty and extensible ChangeLog formatter

```
diff.toString()
Diff:
1. NewObject{globalId:'org.javers.core.model.DummyUserDetails/1'}
2. ValueChange{globalId:'org.javers.core.model.DummyUser/name', property:'sex', oldVal:'FEMALE', newVal:'MALE'}
3. ReferenceChange{globalId:'org.javers.core.model.DummyUser/name', property:'dummyUserDetails', oldRef:'', newRef:'org.javers.core.model.DummyUserDetails/1'}

SimpleTextChangeLog:
    new object: org.javers.core.model.DummyUserDetails/1
  changed object: org.javers.core.model.DummyUser/name
    value changed on 'sex' property: 'FEMALE' -> 'MALE'
    reference changed on 'dummyUserDetails' property: 'null' -> 'org.javers.core.model.DummyUserDetails/1'
```

What is the issue?

**Comment by jrpe99** [comment]
The String property changed from NULL to '' 

toString on  ValueChange displayed  property:'internAnmarkningstext', oldVal:'', newVal:'' 
as if the property changed from empty String to empty String, when it changed from null to empty String

From what I can see, processChangeList displayed 'internAnmarkningstext' property: 'null' -> ''
which is correct NULL to empty String

Maybe I made a misstake som where, so if you cannot reproduce just close the issue.

**Comment by jrpe99** [comment]
Sorry, I pressed the wrong button. Was just going to comment ...

**Comment by bartoszwalacik** [comment]
ok, now I see the difference:

```
diff.toString()
Diff:
1. ValueChange{globalId:'org.javers.core.model.DummyUser/name', property:'surname', oldVal:'', newVal:'jon'}

SimpleTextChangeLog:
  changed object: org.javers.core.model.DummyUser/name
    value changed on 'surname' property: 'null' -> 'jon'
```

So, in toString() we should display 'null' instead of ''

**Comment by bartoszwalacik** [comment]
fix is in master, will be released in 3.5.2
